Tests/TuistDocIntegrationTests/TuistDocIntegrationTests.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import TSCBasic
+import TuistSupport
+import XCTest
+
+@testable import TuistDoc
+@testable import TuistSupportTesting
+
+final class TuistDocIntegrationTests: TuistTestCase {
```

Comment:
The name of this test and file should be `SwiftDocControllerIntegrationTests`.

Reply:
I'd remove this test entirely and rather implement an acceptance test that covers the documentation workflow.